client/lib/preferences/actions.js

```diff
@@ -36,7 +37,12 @@ function mergePreferencesToLocalStorage( preferences ) {
 }
 
 PreferencesActions.fetch = function() {
-	var localStorage = getLocalStorage();
+	var localStorage = getLocalStorage(),
+		userIsLoggedIn = Boolean( user.data );
```

Comment:
Can we use the `get()` accessor here like we do [in signup](https://github.com/Automattic/wp-calypso/blob/master/client/signup/main.jsx#L149) or is there a reason not to do this?